scripts/variantstore/wdl/extract/get_columns_for_import.py

```diff
@@ -0,0 +1,253 @@
+import requests
+import argparse
+
+from terra_notebook_utils import table
+import re
+
+
+maxNumSamples = 50
+
+
+def get_sample_sets(workspace_namespace, workspace_name):
```

Comment:
Assuming this is here in preparation for the next talk, where we make this work with sample sets?

Just checking, as it's otherwise unused.  No need to remove it from this PR either way, though